ui/src/views/Documentation/Reference/SchemaIndex.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import { withRouter } from 'react-router-dom';
+import sortBy from 'lodash/sortBy';
+import filter from 'lodash/filter';
+import Entry from './Entry';
+import HeaderWithAnchor from '../components/HeaderWithAnchor';
+import references from '../../../../../generated/references.json';
+
+@withRouter
```

Comment:
Is this needed?

Reply:
No, I don't this its getting used, removing it.